### PR TITLE
✅ E2E: Get EXA

### LINKED ISSUE #1223: [get exa] same chain e2e test

### PR DESCRIPTION
**Pull request by jgalat**
closes #1223 

This adds a suit of tests for the `Get EXA` feature.
- Swap ETH for EXA on OP
- Swap OP for EXA on OP

### CODE REVIEW

File: e2e/fixture/socket.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      name: 'Optimism',
+      symbol: 'OP',
+      decimals: 18,
+      chainAgnosticId: null,
+      icon: 'https://optimistic.etherscan.io/token/images/optimism_32.png',
+      logoURI: 'https://optimistic.etherscan.io/token/images/optimism_32.png',
+    },
+  };
+
+  type Balances = {
```
Comment (sebipap) [inline e2e/fixture/socket.ts:30]: minor: we generally call types in singular, so `type Balance` and instance it using `Balance[]`

File: e2e/fixture/socket.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    account: Address;
+    balances: {
+      symbol: 'ETH' | 'OP';
+      amount: number;
+    }[];
+  };
+
+  const balances = async (params: Balances) => {
+    await page.route(/api\.socket\.tech\/v2\/balances/, async (route) => {
```
Comment (sebipap) [inline e2e/fixture/socket.ts:39]: minor: why do we use a regex here? couldn't we just use a template string?

**Review by sebipap — APPROVED** [review]
✅

File: e2e/fixture/socket.ts (hunk 2 above)
Comment (jgalat) [reply e2e/fixture/socket.ts:39]: Couldn't make it work, think it has to be the exact string which is filled with query parameters

**Review by franm91 — APPROVED** [review]

**Pull request merged**